### DISCUSSION TITLE
画像と背景の境目がわかるようにしてほしい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#213] [要望・アイデア・不具合]
白背景ベースの画像を貼り付けると、Qiita本文の背景と同色になってしまい、どこまでが画像なのかわからなくなります。

<img width="855" alt="Screen Shot 2022-05-04 at 16 48 09" src="https://user-images.githubusercontent.com/1148320/166641743-2d34b2f1-2ac6-4c12-95f1-8edd2f646bcd.png">

たとえば以下のように薄い枠線を付けて、どこまでが画像なのか明確にすると、見た目上メリハリが付くと思うのですが、いかがでしょうか？

<img width="855" alt="Screen Shot 2022-05-04 at 16 48 09 3" src="https://user-images.githubusercontent.com/1148320/166641839-30fdc8fa-b45b-4207-b66e-58668ed07315.png">

ちなみに上記画像は僕のQiita記事からの抜粋です。

[Ruby初心者がGitHub上でよくやりがちな誤爆メンションとその解決策 \- Qiita](https://qiita.com/jnchito/items/5473e80cc6b34cd5dab1)

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご意見いただきありがとうございます！
こちら確かに境界がわかりにくいと思いますので、対応いたします🙏

**Reply by JunichiIto** [reply]
@tanaka-asami415 ありがとうございます。よろしくお願いします！

**Reply by kyntk** [reply]
@JunichiIto 大変お待たせしました！
記事内の画像に枠がつくように変更しました！
https://qiita.com/release-notes#2022-06-02

**Reply by JunichiIto** [reply]
境目がハッキリして見やすくなりました 👍 
ご対応どうもありがとうございました！
